RevenueCatUI/Data/TemplateViewConfiguration.swift

```diff
@@ -62,13 +62,8 @@ extension TemplateViewConfiguration.PackageConfiguration {
         switch self {
         case let .single(package):
             return package
-        case let .multiple(packages):
-            guard let package = packages.first else {
-                // `create()` makes this impossible.
-                fatalError("Unexpectedly found no packages in `PackageConfiguration.multiple`")
-            }
-
-            return package
+        case let .multiple(first, _):
```

Comment:
Thanks to this new definition we don't need a `fatalError`, we know at compile time that it's not empty 😎